packages/core/src/schema/models/connectedEvaluations.ts

```diff
@@ -0,0 +1,38 @@
+import { relations } from 'drizzle-orm'
+import { bigint, primaryKey, uuid } from 'drizzle-orm/pg-core'
+
+import { documentVersions, latitudeSchema } from '..'
+import { timestamps } from '../schemaHelpers'
+import { evaluations } from './evaluations'
+
+export const connectedEvaluations = latitudeSchema.table(
+  'connected_evaluations',
+  {
+    documentUuid: uuid('document_uuid')
+      .notNull()
+      .references(() => documentVersions.documentUuid),
+    evaluationId: bigint('evaluation_id', { mode: 'number' })
+      .notNull()
+      .references(() => evaluations.id),
+    ...timestamps(),
+  },
+  (table) => ({
+    connectedEvaluationsPk: primaryKey({
+      columns: [table.evaluationId, table.documentUuid],
+    }),
+  }),
+)
```

Comment:
**Connected evaluations** — Relation between a document and an evaluation. Defines which evaluations are connected to which documents, independently of document's version.

Not much to say here. The `documentUuid` foreign key may not be possible since it is not a primary key from the `documentVersion` table

Reply:
Add id primary key to the table. I know you have the compound pk but having a dedicated autoincremental column facilitates working with the table directly when running data migrations, debugging prod db issues, etc..

Reply:
Add also indexes for document uuid and relation is columns

Reply:
hmm, I'll add it, but I don't actually see the need for it. What advantages does it have over the composed pk?

Reply:
When working with tables directly, it is much easier to work with a single colum id number than having to work with two columns ids to find stuff. I know it seems a small thing but it becomes a pita when you are debugging production issues for a user for a whole day.